packages/date/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.2.0 (2018-10-24)
```

Comment:
```suggestion
## 2.2.0 (unreleased)
```

Reply:
The date is added on release, see https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md#maintaining-changelogs

Reply:
Thank you, with this PR package updates and deprecation are lot clear to me :)